CMakeLists.txt

```
@@ -1159,6 +1159,70 @@ ExternalProject_Add( fmt
 
 list(APPEND build_list fmt)
 
+
+################################
+# Fesapi
+################################
+
+if( ENABLE_VTK AND ENABLE_FESAPI )
+
+    #
+    # First ENABLE Minizip
+    #
+    set(MINIZIP_DIR "${CMAKE_INSTALL_PREFIX}/minizip")
+    set(MINIZIP_URL "${TPL_MIRROR_DIR}/Minizip-1.1_cmake0.1.tar.gz")
+    message(STATUS "Building Minizip found at ${MINIZIP_URL}")
+
+    ExternalProject_Add( minizip
+                        URL ${MINIZIP_URL}
+                        PREFIX ${PROJECT_BINARY_DIR}/minizip
+                        INSTALL_DIR ${MINIZIP_DIR}
+                        BUILD_COMMAND make -j ${NUM_PROC}
+                        INSTALL_COMMAND make install
+                        CMAKE_ARGS -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+                                -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+                                -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
+                                -DCMAKE_CXX_FLAGS=${TPL_CXX_STANDARD}                                
+                                -DCMAKE_PREFIX_PATH:PATH=<INSTALL_DIR>
+                                )
+
+    list(APPEND build_list minizip)
+
+    #
+    # Then Fesapi
+    #
+    set(FESAPI_DIR "${CMAKE_INSTALL_PREFIX}/fesapi")
+    set(FESAPI_URL "${TPL_MIRROR_DIR}/fesapi-2.3.0.0.tar.gz")
+    message(STATUS "Building Fesapi found at ${FESAPI_URL}")
+
+    ExternalProject_Add( fesapi
+                        URL ${FESAPI_URL}
+                        PREFIX ${PROJECT_BINARY_DIR}/fesapi
+                        INSTALL_DIR ${FESAPI_DIR}
+                        BUILD_COMMAND make -j ${NUM_PROC}
+                        INSTALL_COMMAND make install
+                        DEPENDS ${HDF5_DEPENDENCIES} minizip
```

To be honest I don't know why we used `HDF5_DEPENDENCIES` for `silo` and `conduit` targets as opposed to just writing `DEPENDS hdf5`. HDF5 a non-optional dependency and that var always get set to just `hdf5`. Perhaps there were multiple targets to depend on at some point but that got changed later? Up to you if you want to keep it this way for consistency, or change to
```suggestion
                        DEPENDS hdf5 minizip
```